explorer/src/lib/dusk/components/dusk.components.d.ts

```diff
@@ -1,4 +1,4 @@
-type BadgeVariant = "neutral" | "success" | "warning" | "error";
+type BadgeVariant = "neutral" | "success" | "warning" | "error" | "alt";
```

Comment:
I'm not going to request this be changed right now, however, we need to be mindful of the variants and attempt to standardise these within Duskit.

Indeed, it might be better to split the concept of state out of the concept of variation, or have variants be a something separate to states entirely.

Something to bear in mind ;)

Reply:
Yeah, this doesn't seem exactly right, as it might be possible to have an alt appearance of the rest of the variants too...